src/test/java/org/takes/facets/auth/social/PsGoogleTest.java

```diff
@@ -112,18 +138,7 @@ public Response act(final Request req) throws IOException {
                             Matchers.containsString(PsGoogleTest.GOOGLE_TOKEN)
                         );
                         return new RsJson(
-                            Json.createObjectBuilder()
-                                .add("displayName", "octocat")
-                                .add("id", "1")
-                                .add(
-                                    "image",
-                                    Json.createObjectBuilder()
-                                        .add(
-                                            "url",
-                                            PsGoogleTest.AVATAR
-                                        )
-                                )
-                                .build()
+                            jsonBuilder("1").add("displayName", octocat).build()
```

Comment:
@dalifreire sorry, this suggestion was a mistake. could you revert the changes done to this class in commit https://github.com/yegor256/takes/pull/689/commits/b272c20249759b8939503b4ed70b4bc17b36a87e ?